cmd/operator-sdk/new/cmd.go

```diff
@@ -35,7 +35,9 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client/config"
 )
 
-func NewCmd() *cobra.Command {
+func NewCmd() *cobra.Command { //nolint:golint
+	// The nolint here is used to hide the warning "func name will be used as new.NewCmd by other packages, and that stutters; consider calling this Cmd"
+	// which is a false positive.
```

Comment:
can we wrap the comment so it's 80 columns per line? At least break up the first one into 2 lines.